posthog/helpers/session_recording.py

```diff
@@ -63,6 +69,9 @@ def compress_and_chunk_snapshots(events: List[Event], chunk_size=512 * 1024) ->
                 },
             },
         }
+        if window_id:
```

Comment:
This kind of smells of incremental design and makes the code harder to follow.

Why not always set window_id above as we do with everything else, just accept the nullability